public/app/config/collections/all/searchInputFields.js

```diff
@@ -0,0 +1,89 @@
+(function () {
```

Comment:
This file used to be called `searchTypes` but this name is better, it describes the input fields for a given search form.  This includes the basic search and the advanced search.  The basic search options have `basicSearch` set to `true` (and a `glyphicon`), others are only displayed in advanced search.